oat-sa/tao-test-runner-qti-fe
# Fix/tca 483/review tabs incorrectly announced

**Related to**
https://oat-sa.atlassian.net/browse/TCA-483

**Description**
KB actions for All / Unanswered / Flagged tabs are incorrectly announced

**How to test**
1. as a user start a test
2. enable JAWS
3. navigate to All / Unanswered / Flagged tabs
4. set focus to each tab, listen to SR's announcements

It should be no announcement of navigation via Arrow keys
